src/NativeScript/ObjC/Constructor/ObjCConstructorBase.mm

```diff
@@ -132,9 +150,23 @@ static void writeAdapter(ExecState* execState, const JSValue& value, void* buffe
 
 const WTF::Vector<WriteBarrier<ObjCConstructorCall>>& ObjCConstructorBase::initializers(VM& vm, GlobalObject* globalObject) {
     if (this->_initializers.size() == 0) {
-        for (ObjCConstructorCall* initializer : this->generateInitializers(vm, globalObject, this->klass())) {
-            this->_initializers.append(WriteBarrier<ObjCConstructorCall>(vm, this, initializer));
-        }
+        JSC::DeferGCForAWhile deferGC(vm.heap);
+        const Metadata::InterfaceMeta* metadata = this->metadata();
+
+        do {
+            std::vector<const Metadata::MethodMeta*> initializers = metadata->initializersWithProtcols();
+            for (std::vector<const Metadata::MethodMeta*>::iterator init = initializers.begin(); init != initializers.end(); ++init) {
```

Comment:
You can use `for (const Metadata::MethodMeta* method : initializers)` if you want.